05week/account.js

```diff
@@ -0,0 +1,69 @@
+'use strict'
+
+class Account {
```

Comment:
I was hoping to see a deposit(amt) method, that would auto fill the payee as "deposit"